src/Simplex/Chat/Options.hs

```diff
@@ -123,18 +129,36 @@ coreChatOptsP appDir defaultDbFileName = do
   socksProxy <-
     flag' (Just defaultSocksProxy) (short 'x' <> help "Use local SOCKS5 proxy at :9050")
       <|> option
-        parseSocksProxy
+        strParse
         ( long "socks-proxy"
             <> metavar "SOCKS5"
             <> help "Use SOCKS5 proxy at `ipv4:port` or `:port`"
             <> value Nothing
         )
+  smpProxyMode_ <-
+    optional $
+      option
+        textJsonParse
+        ( short 'y'
```

Comment:
Maybe remove short options? This is usually written once then repeated. y is often used as "automatically answer yes"